lib/em-jack/connection.rb

```diff
@@ -284,6 +284,11 @@ def disconnect
       @close_df = EM::DefaultDeferrable.new
     end
 
+    def force_reconnect
```

Comment:
Confusingly, by Ruby conventions, `force_reconnect` and `reconnect!` read as more or less the same name. What is `reconnect!` for, and if it doesn't serve your purpose, is that cos it's buggy rather than because it wasn't meant to?

Reply:
I was aware there was overlap there, I just wasn't sure whether anything else depended on the behaviour of that `reconnect!` method. I suspect `force_reconnect` may be a better alternative though so should probably just replace it and see if anything breaks 🙈